src/common/containers/landing.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { Anchor } from '../components/button';
+
+import styles from './container.css';
+
+class Landing extends Component {
+  componentWillMount() {
+    if (this.props.auth.authenticated) {
```

Comment:
This is among the reasons I'm not a fan of the separate `/repos` route; at best it just seems to move complexity around.  As I mentioned in the call this morning, I'd prefer to have them both in `/` seeing as we need to test whether the user is authenticated anyway.

Matthew said that they were trying to prepare for one user being able to see another user's dashboard in the future, which would necessitate it being on `/:user` or similar.

Also, without a complete URL reorganisation, we can't use `/repos`, since there exists a GitHub user called "repos".